packages/forms-web-app/src/views/full-planning/out-of-time-shutter-page.njk

```diff
@@ -0,0 +1,17 @@
+{% extends "layouts/main-no-back-link.njk" %}
+
+{% set title="You cannot appeal - Before you start - Appeal a planning decision - GOV.UK" %}
+
+{% block pageTitle %}
+  {{ title }}
+{% endblock %}
+
+{% block content %}
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+      <h1 class="govuk-heading-l">You cannot appeal.</h1>
+      <p class="govuk-body">Your deadline to appeal has passed</p>
+      <p class="govuk-body">Your deadline was 10 October 2021. This is 6 months from the decision date.</a>.<p>
```

Comment:
There are 2 full stops here and why do we have </a> tag? Also, can there be a way for me to uniquely identify this element? May be a data-cy tag?

Reply:
resolved.